include/openPMD/RecordComponent.hpp

```diff
@@ -75,6 +75,57 @@ struct IsContiguousContainer< std::array< T_Value, N > >
 };
 } // namespace traits
 
+/**
+ * @brief Subset of C++20 std::span class template.
+ *
+ * Any existing member behaves equivalently to those documented here:
+ * https://en.cppreference.com/w/cpp/container/span
+ */
+template< typename T >
```

Comment:
Let's put this in its own header file, please :)

Reply:
done